extensions/wikia/CommunityPage/templates/allAdmins.mustache

```diff
@@ -2,9 +2,11 @@
 <ul class="reset top-contributors">
 	{{#allAdminsList}}
 		<li class="community-page-contributors-list-item">
-			<div class="badge-container">{{{badge}}}</div>
-			<div class="avatar-container">
-				<a data-tracking="all-admins-user-avatar" href="{{profilePage}}">{{{avatar}}}</a>
+			<div>
```

Comment:
Can all of those newly added wrapper divs have a informative class on them, i.e. user-container or something similar?